particles/Tutorial/Kotlin/1_HelloWorld/HelloWorld.arcs

```diff
@@ -1,5 +1,7 @@
 // Tutorial: Hello World
 // Defines a really simple particle that just prints out "Hello, world!"
+meta
```

Comment:
I think you may need to update the tutorial text to explain why we need a `meta` section. 

(You may do this later in the PR)

((This also can be deferred until after march))